core/dbt/parser/manifest.py

```diff
@@ -1378,40 +1397,40 @@ def _process_refs_for_node(manifest: Manifest, current_project: str, node: Manif
 def _process_semantic_information_for_node(
     manifest: Manifest, current_project: str, node: ManifestNode
 ):
-    """ Given a manifest and a node in that manifest, process all of the semantic 
+    """Given a manifest and a node in that manifest, process all of the semantic
     information for the related nodes. This occurs in multiple steps:
       1. If the node is a seed or model and is_public is true, continue with compilation. Otherwise ignore.
       2. Loop through all of the models listed in relationships and check if they also
       have is_public set to true. If not, append to a list and then raise that list in a CompilationException.
-      3. Loop through the columns where is_primary_key is set to true and then create a 
+      3. Loop through the columns where is_primary_key is set to true and then create a
       composite list at the model level of those columns
       4. Create the inverse_relationship of each relationship and validate it against the other model.
       If it doesn't exist, add it. If a relationship exists with the same name, confirm that the properties match.
       If the properties don't match, raise a CompilationException.
-      
+
     """
     target_model_package: Optional[str] = None
-    
+
     # Limit this parsing to public models/seeds that have relationships
-    if node.resource_type == "model" and node.is_public == True:
+    if node.resource_type == "model" and node.is_public:
```

Comment:
👍

Reply:
this was the result of the `flake8` pre-hook!